colossalai/utils/moe.py

```diff
@@ -0,0 +1,50 @@
+import torch.nn as nn
+import torch.distributed as dist
+from colossalai.core import global_context as gpc, moe_context as moe_env
+from colossalai.context import ParallelMode
+from .common import is_using_ddp
+
+
+def get_moe_param_dict(model: nn.Module):

```

Comment:
-> dict[int, List[torch.nn.Parameters]]
develop a good habits of writing return value type...

Reply:
rename it as get_moe_epsize_param_dict

Reply:
done